modules/extension/xsd/xsd-fes/src/test/java/org/geotools/filter/v2_0/bindings/DWithinBindingTest.java

```diff
@@ -1,16 +1,83 @@
 package org.geotools.filter.v2_0.bindings;
 
+import static java.lang.String.format;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import javax.xml.parsers.ParserConfigurationException;
+import org.geotools.filter.FilterFactoryImpl;
+import org.geotools.filter.spatial.DWithinImpl;
 import org.geotools.filter.v1_1.FilterMockData;
 import org.geotools.filter.v2_0.FES;
+import org.geotools.filter.v2_0.FESConfiguration;
 import org.geotools.filter.v2_0.FESTestSupport;
+import org.geotools.xml.Configuration;
+import org.geotools.xml.Encoder;
+import org.geotools.xml.Parser;
+import org.junit.Test;
+import org.opengis.filter.Filter;
+import org.opengis.filter.FilterFactory;
 import org.opengis.filter.spatial.DWithin;
-import org.w3c.dom.Document;
+import org.xml.sax.SAXException;
+
+public class DWithinBindingTest {
+
+    private static final Configuration CONFIGURATION = new FESConfiguration();
+
+    private static final Parser PARSER = new Parser(CONFIGURATION);
 
-public class DWithinBindingTest extends FESTestSupport {
+    private static final Encoder ENCODER = new Encoder(CONFIGURATION);
 
+    // TODO: Nothing being asserted - remove?
```

Comment:
Seems like this test wasn't asserting anything. Remove?

Reply:
Yep, the new tests are covering encoding, so agree it should be removed.